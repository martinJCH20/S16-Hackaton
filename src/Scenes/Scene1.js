import React, { Component } from 'react'
import { View, StyleSheet, Text, ScrollView} from 'react-native'

export default class Scene1 extends Component {
  constructor(props){
    super(props)
    this.state = {
      cadena:[],
        types: []
    }
  }  
  goToScene2 = (item, index, imagePokemon, type  ) => {
    //console.warn('item', item)
    this.props.navigation.navigate('Scene2', {item: item, index: index + 1, img: imagePokemon, type: type})
  }
    render(){
        const { cadena, types } = this.state;    
         return (
            <View style={{ flex: 1, marginTop: 5}}>
              <Text>Bienvenido</Text>
            </View>
          )
    }
}