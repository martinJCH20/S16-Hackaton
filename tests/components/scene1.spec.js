import React from 'react';
import {Text, View, ImageBackground} from 'react-native';
// import {shallow} from 'enzyme';
import Scene1 from '../../src/Scenes/Scene1';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(<Scene1><Text>Hola</Text></Scene1>).toJSON();
  expect(tree).toMatchSnapshot();
});