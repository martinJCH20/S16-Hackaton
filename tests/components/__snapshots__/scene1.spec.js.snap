// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "marginTop": 5,
    }
  }
>
  <Text>
    Scene 1
  </Text>
</View>
`;
